### DISCUSSION TITLE
获取在线仓库数据失败, 请尝试在通用设置中设置其他更新源, 然后再试一次

### DISCUSSION THREAD

**Discussion by ghost** [the1812/Bilibili-Evolved#2614] [提问 / Q&A]
v2里更换了两个源在线仓库那里都会报错
错误
获取在线仓库数据失败, 请尝试在通用设置中设置其他更新源, 然后再试一次

另外v1里的黑夜模式会考虑修吗？

**Comment by the1812** [comment]
是不是用的 Tampermonkey beta?

> 另外v1里的黑夜模式会考虑修吗？

v1 还不好说, b 站这波改动看起来不小, 很可能我要直接发 v2 正式版了

**Reply by ghost** [reply]
对，用的的确是Tampermonkey beta

**Reply by the1812** [reply]
beta 现在有 bug, 等作者更新 4.17.6149 版
